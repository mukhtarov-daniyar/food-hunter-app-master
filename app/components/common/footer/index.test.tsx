import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './index';

it('renders correctly', () => {
  const tree = renderer.create(<Footer children={''} />).toJSON();
  expect(tree).toMatchSnapshot();
});
