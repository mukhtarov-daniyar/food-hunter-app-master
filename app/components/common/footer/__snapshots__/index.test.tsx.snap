// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "#ffffff",
        "borderTopColor": "#F1F4F9",
        "borderTopWidth": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "paddingHorizontal": 38.46153846153846,
        "paddingVertical": 28.846153846153847,
      },
      undefined,
    ]
  }
/>
`;
